File: modules/chat-core/src/main/java/com/shterneregen/securelan/chat/protocol/WireMessageType.java
package com.shterneregen.securelan.chat.protocol;

public enum WireMessageType {
    HELLO,
    SERVER_KEY,
    CLIENT_KEY,
    ACCEPTED,
    REJECTED,
    CHAT,
    SYSTEM,
    USER_JOINED,
    USER_LEFT,
    SIGNAL,
    DISCONNECT
}

File: modules/chat-core/src/main/kotlin/com/shterneregen/securelan/chat/protocol/WireMessageType.kt
package com.shterneregen.securelan.chat.protocol

enum class WireMessageType {
    HELLO,
    SERVER_KEY,
    CLIENT_KEY,
    ACCEPTED,
    REJECTED,
    CHAT,
    SYSTEM,
    USER_JOINED,
    USER_LEFT,
    SIGNAL,
    DISCONNECT,
}
